pyfolio/plotting.py

```diff
@@ -189,6 +189,7 @@ def plot_rolling_fama_french(returns,
         factor_returns=factor_returns,
         rolling_window=rolling_window)
 
+    rolling_beta = rolling_beta[['SMB', 'HML', 'Mom   ']]
```

Comment:
Just to make sure, those spaces are actually in the column name?

Reply:
yup, the `Mom` column has those spaces in it

Reply:
We should probably fix the spaces.

Reply:
👍

Reply:
So there must be a bad string somewhere.

Reply:
looks like it's in the output of `ep.utils.get_fama_french`, will strip the whitespace from there

Reply:
Should we update it in empyrical then?

Reply:
Yes, this makes it tricky though because there's a version dependency now. Thus, I vote we just drop alpha and side-step the issue (and fix it in empyrical).

Reply:
yup, will fix it in empyrical